templates/{{ cookiecutter.project_slug }}/src/main/java/com/deloitte/controller/{{cookiecutter.resource_name}}Controller.java

```diff
@@ -1,4 +1,4 @@
-package com.deloitte.controllers;
```

Comment:
Revert this changes for now, otherwise we need to to do import changes where ever is controller is used

Reply:
I have checked for implications and made the necessary changes!